Storage/SQL/SQLiteHistory.swift

```diff
@@ -199,19 +237,22 @@ extension SQLiteHistory: BrowserHistory {
     }
 
     public func getSitesByFrecencyWithLimit(limit: Int) -> Deferred<Result<Cursor<Site>>> {
-        return self.getSitesByFrecency(limit: limit, filter: nil)
-    }
+        let groupBy = "GROUP BY \(TableHistory).domain_id "
```

Comment:
I was trying to figure this out for a bit too! But frecency is already based on the grouping (i.e. the total number of visits and time of the last visit for this group). I don't think we need to sum, do we? (We already sum local and remote frecency).

If we switch to desktop's model where each visit gets a frecency of its own, we'd need this to get a composite for all the visits. I have a feeling computing a frecency for each visit will be costly though (still kinda want it).